src/Locale/Translator.php

```diff
@@ -17,6 +17,16 @@ class Translator extends BaseTranslator implements TranslatorContract
 {
     const REFERENCE_REGEX = '/^=>\s*([a-z0-9_\-\.]+)$/i';
 
+    public function get($key, array $replace = [], $locale = null)
```

Comment:
Can we do this the other way around, and have the logic in Laravel's new methods, and have the old methods reference the new ones?

Reply:
The "old" ones come from the Symfony base implementation.

Reply:
The reason I bring this up is that if we eventually want to comply with the laravel contract, we might as well move the logic to laravel's methods. If we want to eventually drop the laravel translation contract, this is fine though.

Reply:
I don't get what you mean, sorry. `trans()` and `transChoice()` are implemented in Symfony's translator, and we don't overwrite these. And since we're mostly/only using Symfony's interface throughout the app, I'd rather keep their implementation as well. These two methods are only needed as a bridge to be able to use this Symfony-based translator in Laravel's validation component.